### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/advanced/testing-websockets.md`

### PR DESCRIPTION
**Pull request by Limsunoh**
Add Korean translation for docs/ko/docs/advanced/testing-websockets.md

### CODE REVIEW

File: docs/ko/docs/advanced/testing-websockets.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+이를 위해 `with` 문에서 `TestClient`를 사용하여 WebSocket에 연결합니다:
+
+```Python hl_lines="27-31"
+{!../../docs_src/app_testing/tutorial002.py!}
+```
+
+/// note | 참고
+
+자세한 내용은 <a href="https://www.starlette.io/testclient/#testing-websocket-sessions" class="external-link" target="_blank">Starlette의 WebSocket 테스트 세션</a>에 관한 설명서를 참고하세요.
```
Comment (kwang1215) [inline docs/ko/docs/advanced/testing-websockets.md:13]:
```suggestion
자세한 내용은 Starlette의 <a href="https://www.starlette.io/testclient/#testing-websocket-sessions" class="external-link" target="_blank"> WebSocket 테스트</a>에 관한 설명서를 참고하세요.
```


How about modifying this part like this?

**Comment by kwang1215** [comment]
LGTM

**Comment by 9zimin9** [comment]
LGTM!

**Review by 9zimin9 — APPROVED** [review]
LGTM !

**Review by kim-sangah — APPROVED** [review]
LGTM👍

**Comment by alejsdev** [comment]
Thanks to everyone for the help! @Limsunoh @kwang1215 @9zimin9 @kim-sangah :grinning: :tada:

**Pull request merged**